src/bench/bench.cpp

```diff
@@ -62,33 +92,39 @@ void benchmark::BenchRunner::RunAll(const Args& args)
     }
 
     std::vector<ankerl::nanobench::Result> benchmarkResults;
-    for (const auto& p : benchmarks()) {
-        if (!std::regex_match(p.first, baseMatch, reFilter)) {
+    for (const auto& [name, bench_func] : benchmarks()) {
```

Comment:
nit: since this unpacking touches a few lines, I think this would benefit from being split out into a separate refactor commit to make that more clear?

Reply:
I think that splitting it would be too much for such a small change.

Reply:
Okay, I don't agree but not that important so you can mark as resolved.